ccd-definition/AuthorisationCaseField/court-admin.json

```diff
@@ -510,6 +510,27 @@
     "UserRole": "caseworker-publiclaw-courtadmin",
     "CRUD": "CRUD"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "singleCareOrder_label",
+    "UserRole": "caseworker-publiclaw-courtadmin",
+    "CRUD": "CRUD"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "multipleCareOrder_label",
+    "UserRole": "caseworker-publiclaw-courtadmin",
+    "CRUD": "CRUD"
+  },
```

Comment:
Do we need delete permissions for the above two labels?

Reply:
yes, we do cleanup of these labels in about to submit